chains/v2/chains_dev.json

```diff
@@ -2510,5 +2510,41 @@
         "color": "linear-gradient(315deg, #007568 31.86%, #91E3DA 100%)",
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Integritee.svg",
         "addressPrefix": 13
+    },
+    {
+        "chainId": "97da7ede98d7bad4e36b4d734b6055425a3be036da2a332ea5a7037656427a21",
```

Comment:
Need to add "parentId" for polkadot parachain it is:
"parentId": "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3",

Reply:
fixed